### DISCUSSION TITLE
Welcome to mini_mini_shell Discussions!

### DISCUSSION THREAD

**Discussion by ntk221** [ntk221/minishell#21] [Announcements]
### ディスカッション機能を追加しました！

この場所は、コーディングに関する疑問点の共有や、設計に関する提案, アイディアなどを気軽に残し、お互いのフィードバックを得ることができる場所として使用できたらいいなと思っています〜🕵️‍♂️

**Comment by nekotonatsu** [comment]
初カキコ☆
